src/SystemCommands-MessageCommands/CmdOtherOptionsMenuGroup.class.st

```diff
@@ -0,0 +1,20 @@
+Class {
+	#name : #CmdOtherOptionsMenuGroup,
```

Comment:
It is too generic name. It's better to rename it as SycOtherRefactoringsMenuGroup.
But I am wonder that we want to have third level submenu. It is very bad UI style